migtests/scripts/live-migration-fallb-run-test.sh

```diff
@@ -30,13 +30,27 @@ else
     source "${TEST_DIR}/env.sh"
 fi
 
+export EXPORT_DIR=${EXPORT_DIR:-"${TEST_DIR}/${NORMALIZED_TEST_NAME}_fallb_export-dir"}
+if [ -n "${SOURCE_DB_SSL_MODE}" ]; then
+  EXPORT_DIR="${EXPORT_DIR}_ssl"
+fi
+
 if [ "${SOURCE_DB_TYPE}" = "oracle" ]
 then
 	source ${SCRIPTS}/${SOURCE_DB_TYPE}/live_env.sh
 else
 	source ${SCRIPTS}/${SOURCE_DB_TYPE}/env.sh
 fi
 
+if [[ "${SOURCE_DB_TYPE}" == "postgresql" || "${SOURCE_DB_TYPE}" == "mysql" ]]; then
+    export SOURCE_DB_NAME=${SOURCE_DB_NAME:-"${NORMALIZED_TEST_NAME}_fallb"}
+elif [[ "${SOURCE_DB_TYPE}" == "oracle" ]]; then
+    export SOURCE_DB_SCHEMA=${SOURCE_DB_SCHEMA:-"${NORMALIZED_TEST_NAME:0:10}_fallb"} && export SOURCE_DB_SCHEMA=${SOURCE_DB_SCHEMA^^}
```

Comment:
why `:0:10` ? pls add a comment. 
what is ^^?

Reply:
Using ^^ to make the schema uppercase